src/Components/Publishing/__stories__/Article.story.tsx

```diff
@@ -70,6 +71,52 @@ ads.forEach(mediaType => {
   })
 })
 
+story.add(`Multiple standard articles`, () => {
+  const article = {
+    ...StandardArticle,
+    sections: [
+      {
+        type: "text",
+        body:
+        "<p>What would Antoine Court?</p>",
+      }
+    ]
+  }
+  return (
+    <div>
+      <Article
+        article={article}
+        display={Display('slideshow')}
+        relatedArticlesForPanel={RelatedPanel}
+        relatedArticlesForCanvas={RelatedCanvas}
+        emailSignupUrl="#"
+      />
+      <Break />
+      <Article
+        article={article}
+        display={Display('video')}
+        relatedArticlesForPanel={RelatedPanel}
+        relatedArticlesForCanvas={RelatedCanvas}
+        emailSignupUrl="#"
+      />
+      <Break />
+      <Article
+        article={article}
+        display={Display('image')}
+        relatedArticlesForPanel={RelatedPanel}
+        relatedArticlesForCanvas={RelatedCanvas}
+        emailSignupUrl="#"
+      />
+    </div>
+  )
+})
+
+const Break = styled.div`
```

Comment:
Grabbed this from force code so we would have a better testbed for seeing how infinite scroll is displayed